rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy import Enum
+from sqlalchemy import ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+
+ModelBase = declarative_base()
+
+
+class Game(ModelBase):
+    __tablename__ = 'game'
+
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    round_id = Column(Integer, ForeignKey('round.id'))
+    rounds_choice = Column(Integer)
+    current_round = Column(Integer)
+    winner_id = Column(Integer, ForeignKey('user.id'))
+
+    player_1 = relationship('User', foreign_keys=[player_1_id])
+    player_2 = relationship('User', foreign_keys=[player_2_id])
+    winner = relationship('User', foreign_keys=[winner_id])
+    round = relationship('Round')
+
+
+class Round(ModelBase):
+    __tablename__ = 'round'
+
+    id = Column(Integer, primary_key=True)
+    player_1_choice_id = Column(Integer, ForeignKey('choice.id'))
+    player_2_choice_id = Column(Integer, ForeignKey('choice.id'))
+    winner_id = Column(Integer, ForeignKey('user.id'))
+    game_id = Column(Integer, ForeignKey('game.id'))
+
+    game = relationship('Game')
+    user = relationship('User')
+    player_1_choice = relationship('Choice', foreign_keys=[player_1_choice_id])
+    player_2_choice = relationship('Choice', foreign_keys=[player_2_choice_id])
+
+
+class User(ModelBase):
+    __tablename__ = 'user'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    score = Column(Integer)
+    choices_id = Column(Integer, ForeignKey('choice.player_id'))
```

Comment:
This FK column is invalid because it would need to point to more than one value (this is the same as my comment regarding `Game.round_id`; `Choice.player_id` establishes the relationship between `Choice` and `User` in this case.